electricitymap/contrib/lib/models/event_lists.py

```diff
@@ -117,20 +117,43 @@ def merge_production_breakdowns(
                 f"Trying to merge production outputs from multiple source types \
                 , got {len(source_types)}: {', '.join(source_types)}"
             )
-        df = df.groupby(level=0, dropna=False).sum(numeric_only=True, min_count=1)
+
+        def aggregate(value: pd.Series) -> Optional[float]:
```

Comment:
I think this needs a union return type of `Union[Float, List, None]` list could probably be further typed if needed.